packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/create-okta-application/index.md

```diff
@@ -13,5 +13,6 @@ Before you can sign a user in, you need to create an Okta app integration that r
 1. Enter a name for your app integration (or leave the default value).
 1. Enter your callback route for the **Login redirect URI**. This is the full redirect URI that you defined in the <GuideLink link="../define-callback/">previous step</GuideLink> (for example, `com.okta.example:/login`).
 1. Enter your callback route for the **Logout redirect URI**. This is the full redirect URI that you defined in the <GuideLink link="../define-callback/">previous step</GuideLink> (for example, `com.okta.example:/logout`).
+1. Include a URI in the **Initiate login URI** box to have Okta initiate the sign-in flow by redirecting to this endpoint (for example, `https://example:0000.com/login`). This triggers the client to send an authorize request. This URI is also used when users reset their passwords and need to be redirected back to your app.
```

Comment:
It should not include mobile, Okta cannot initiate a request to mobile phone.

Reply:
@yuliu-okta and @venkatviswanathan-okta 

Should the tooltip text for this field in the Admin Console specify this field is for Web Apps only then? This is the tooltip text that Venkat and you approved:

Include a URI to have Okta initiate the sign-in flow by redirecting to this endpoint. This triggers the client to send an authorize request.

Should we add that it's for browser-based apps only?

Reply:
@yuliu-okta Do we use this with Native apps today? I see the configuration available for native apps as well.